tests/extract/test_extract_job.py

```diff
@@ -104,22 +103,27 @@ def test_django_cannot_encode_datetime_strings(extract_job_settings):
             list(source.query(sql_query, stream=True, row_tuples=True))
 
 
-def test_extract_job(complex_job, extract_job_settings, now):
+@pytest.mark.freeze_time('2020-07-01', ignore=['mo_threads'])
+def test_extract_job(complex_job, extract_job_settings):
     """
     If you find this test failing, then copy the JSON in the test failure into the test_extract_job.json file,
     then you may use the diff to review the changes.
     """
-    source = MySQL(extract_job_settings.source.database)
-    extractor = MySqlSnowflakeExtractor(extract_job_settings.source)
-    sql = extractor.get_sql(SQL("SELECT " + text(complex_job.id) + " as id"))
+    with MySQL(extract_job_settings.source.database) as source:
+        with MySqlSnowflakeExtractor(extract_job_settings.source) as extractor:
+            sql = extractor.get_sql(SQL("SELECT " + text(complex_job.id) + " as id"))
 
-    acc = []
-    with source.transaction():
-        cursor = list(source.query(sql, stream=True, row_tuples=True))
-        extractor.construct_docs(cursor, acc.append, False)
+            acc = []
+            with source.transaction():
+                cursor = list(source.query(sql, stream=True, row_tuples=True))
+                extractor.construct_docs(cursor, acc.append, False)
 
-    doc = acc[0]
-    doc.guid = complex_job.guid
+    doc = first(acc)
+    doc.guid = first(JOB).guid  # NEW EACH TIME
+
+    job_guid = first(jx.drill(JOB, "job_log.failure_line.job_guid"))
+    for fl in jx.drill(doc, "job_log.failure_line"):
+        fl.job_guid = job_guid
```

Comment:
`job_log.failure_line.job_guid` are unique each time, so we set them to what's in the json file